src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -759,6 +764,80 @@ public void connect(String portName, int busIndex0, ModuleInst other, String oth
         }
     }
 
+    /**
+     * Connects unconnected SitePinInst inputs on the module instance to either GND
+     * or VCC (some reset input use VCC as an input and use the inverter to drive
+     * GND).
+     */
+    public void tieOffUnconnectedInputs(boolean verbose) {
+        for (Port port : getModule().getPorts()) {
+            if (port.isOutPort())
+                continue;
+
+            for (SitePinInst pin : getCorrespondingPins(port)) {
+                if (pin == null) {
+                    System.err.println("ERROR: Problem encountered with finding corresponding SitePinInst on port "
+                            + getName() + "/" + port.getName() + " skipping...");
+                }
+                if (pin != null && (pin.getNet() == null || pin.getNet().getSource() == null)) {
+                    // If a port has a RST inverter, drive it with VCC to get GND
+                    boolean connectToVCC = pin.getName().contains("RST");
+                    if (verbose) {
+                        System.out.println(getName() + "/" + port.getName() + " SitePinInst=[" + pin
+                                + "] has no source, connecting to " + (connectToVCC ? "VCC" : "GND"));
+                    }
+
+                    connectToStaticNet(port.getName(), connectToVCC);
```

Comment:
Do we know if simply connecting the SPI to VCC does what we expect? Does any intra-site routing/SitePIPs need to be set?
EDIT: Perhaps not relevant if we stop inverting here.